### PR TITLE
WIP: Move to session id instead of callsign crc check

### PR DESCRIPTION
**Pull request by DJ2LS**
We want to move to session id instead of a callsign crc check during an arq transmission with the goal of saving payload.

- [x] Move to session id
- [x] Move payload to first bytes of signalling frame - lets keep the end of payload free
- [ ] Passing ctests
- [ ] Use this PR for fixing timeout issues
- [x] Add an additional decoding signalling mode in case we need one later ( different mode between ACK and ARQ INIT )
- [x] Rename decoding modes to their function instead of mode name
- [x] Testing Testing Testing
- [x] --Fix ARQ-Session-Mode
- [x] --Improve ARQ Transmission Stopping
- [x] Increase protocol version

Also added:

- [x] Fix of Scatter
- [x] RMS for audio level

### CODE REVIEW

**Comment by DJ2LS** [comment]
@kronenpj I closed the other PR with branch "ls-session-id". This one is more clean - however I have no idea why GUI changes appear here....

**Comment by kronenpj** [comment]
One concern I have is that, in the US, we are required to supply a callsign at the conclusion of a conversation. The two-byte disconnect packet no longer contains that information. Unless we force an ID packet, this application will no longer be compliant with US rules.

**Review by DJ2LS — COMMENTED** [review]
from my side we need some cleanup, but the code seems to be working. So we might need to gather some OTA experience, now.

File: tnc/data_handler.py [hunk 1]
```diff
@@ -2190,17 +2281,17 @@ def transmit_qrv(self) -> None:
...
         qrv_frame[7:11] = helpers.encode_grid(static.MYGRID.decode("UTF-8"))
 
         self.log.info("[TNC] ENABLE FSK", state=static.ENABLE_FSK)
 
         if static.ENABLE_FSK:
-            self.enqueue_frame_for_tx(qrv_frame, c2_mode=FREEDV_MODE.fsk_ldpc_0.value)
+            self.enqueue_frame_for_tx([qrv_frame], c2_mode=FREEDV_MODE.fsk_ldpc_0.value)
         else:
-            self.enqueue_frame_for_tx(qrv_frame)
+            self.enqueue_frame_for_tx([qrv_frame], c2_mode=FREEDV_MODE.datac0.value, copies=1, repeat_delay=0)
```
[the remark this answers is not in the document]
Comment (DJ2LS) [reply tnc/data_handler.py:2292]: solved partially

**Pull request merged**